arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -217,5 +217,26 @@ public async Task GetCollectionCountAsync_ShouldSucceed()
             }));
             Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
         }
+
+        [Fact]
+        public async Task RenameCollectionAsync_ShouldSucceed()
+        {
+            var response = await _collectionApi.RenameCollectionAsync(_testCollection, new RenameCollectionRequest
+            {
+                Name = "testingCollection"
+            });
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.Equal("testingCollection", response.Name);
+            Assert.False(response.IsSystem);
+            Assert.NotNull(response.Id);
+            Assert.False(response.Error);
+            // set the test collection back to the constant
```

Comment:
Tests might run in parallel, for that reason I think we should not rename a collection that might be used in other tests. Can we create a separate collection specifically for this test?  Then we don't have to worry about putting it back.